src/components/ChannelPreview.tsx

```diff
@@ -13,28 +13,24 @@ type ChannelPreviewProps = {
 }
 
 export const ChannelPreview: React.FC<ChannelPreviewProps> = ({ id, className, onClick }) => {
+  const [{ url }, getAssetUrl] = useAsset()
   const { channel, loading } = useChannel(id ?? '', { fetchPolicy: 'cache-first', skip: !id })
   const { videoCount } = useChannelVideoCount(id ?? '', {
     fetchPolicy: 'cache-first',
     skip: !id,
   })
   const isLoading = loading || id === undefined
-  const { getAssetUrl } = useAsset()
+  getAssetUrl(channel?.avatarPhotoAvailability, channel?.avatarPhotoUrls, channel?.avatarPhotoDataObject)
```

Comment:
Two things here:
1. Do we actually need to call that? I think we could just pass those arguments directly into the hook? 
2. I think we still should use the pattern with `entity` and `type`, so we don't need to pass all the necessary fields explicitly

What do you think?